compute_row.py

```diff
@@ -4,10 +4,38 @@
 This allows for distributed processing'''
 
 import clustering,argparse,sys
+from rdkit.Chem import AllChem as Chem
+from rdkit.Chem import AllChem
+from rdkit.DataStructs import FingerprintSimilarity as fs
+from rdkit.Chem.Fingerprints import FingerprintMols
+
+def compute_ligand_similarity(smiles, pair):
```

Comment:
this is duplicative of code in clustering.py, correct?  There should be only one function, which gets imported.

Reply:
It is not. There is differing order of the input file, and the function in clustering creates a dictionary of target -> smile from the files present to the input file of clustering (which has a different format, and would not be applicable to the input to this script), and then (not in parallel) constructs the matrix. In this script, the SMILES are already present in the input file, and we only want to return 1 number, rather than clustering.py's return of an entire matrix.